tutorials/pages/bridge-usdc-testnet-tokens.md

```diff
@@ -0,0 +1,104 @@
+
+![Multichain Select](/select-chain-multichain.png 'Select Chain on Multichain')
+
+![Select USDC](/select-usdc.png 'Select USDC')
+
+![Add USDC to Metamask](/add-usdc.png 'Add USDC to Metamask')
+
+If the previous steps were successful, your wallet should show balances for GoerliETH and USDC when connected to the Goerli network and SMR when connected to ShimmerEVM Testnet.
+
+![USDC in Metamask](/usdc-metamask.png 'USDC in Metamask')
+
+![SMR Balance in Metamask](/smr-balance-metamask.png 'SMR Balance in Metamask')
+
+Now you're ready to bridge USDC tokens over to ShimmerEVM.
+
+## Bridge USDC from Goerli to ShimmerEVM Testnet
+
+Go to our [Token Bridge](https://iota-bridge.vercel.app/) and connect your Metmask wallet to the site. Select `Goerli` as the source network, `USDC` as the token and `ShimmerEVM Testnet` as the target network. Then you can enter the amount of USDC tokens you want to bridge. When all parmameters have been set, you're ready to `Transfer`.
+
+**_NOTE:_** The difference between the `Send` and `Receive` amounts are the fees you will pay to the bridge operator, in this case Multichain.
+
+![Bridge Assets](/bridge-assets.png 'Bridge Assets')
+
+The page will summarize your selections and ask you to confirm the transaction.
+
+![Confirmation on Metamask](/confirm-swap.png 'Confirmation on Metamask')
+
+Metamask will ask you to set, review and confirm the max amount of USDC token that may be processed by the bridge contract. This is a safety measure for users of a particular contract and puts a cap on the max amount of tokens the contract gains control over. This will pop up again whenever your set spending cap was reached. Afterwards you can confirm the actual transaction.
+
+![Spending cap on Metamask](/metamask-spending-cap.png 'Spending cap on Metamask')
+
+![Spending cap review on Metamask](/metamask-spending-cap-review.png 'Spending cap review on Metamask')
+
+![Transaction on Metamask](/metamask-tx.png 'Transaction on Metamask')
+
+After the transaction was confirmed on Metamask, you can follow the status of the bridge transfers on the Token Bridge page. The USDC tokens will be sent to a Multichain contract on Goerli, routed towards a Multichain contract on ShimmerEVM Testnet and then sent to your target address there.
+
+![Transaction on Multichain](/multichain-tx.png 'Transaction on Multichain')
+
+![Transaction on Multichain Sent](/multichain-tx-sent.png 'Transaction on Multichain Sent')
+
+## Check USDC balance on ShimmerEVM Testnet
+
+Now you can connect your Metmask wallet to the ShimmerEVM Testnet and check if the USDC tokens have arrived successfully. Since you have already added the USDC token contract on ShimmerEVM Testnet to your Metamask wallet, you should now see the USDC token balance there.
+
+<p style="text-align: center">
+    <kbd>
+        <img src="/metamask-usdc-balance.png"  width="300">
+    </kbd>
+</p>
+
+![USDC Balance on Metamask](/metamask-usdc-balance.png 'USDC Balance on Metamask')
```

Comment:
```suggestion
Now you can connect your Metmask wallet to the ShimmerEVM Testnet and check if the USDC tokens have arrived successfully. Since you have already added the USDC token contract on ShimmerEVM Testnet to your Metamask wallet, you should now see the USDC token balance there.

![USDC Balance on Metamask](/metamask-usdc-balance.png 'USDC Balance on Metamask')
```